views/users/new.pug

```diff
@@ -0,0 +1,36 @@
+
+extends ../application.pug
+
+block content
+	.container
+		.row
+			.col-xs-6.offset-xs-2
+				if errors
+					each error in errors
+						div(class="alert alert-danger" role="alert")
+							span= error.message
+				br
+				h1 Create a New User:
+				hr
+				form(action="/register" method="POST")
+					div(class="form-group")
+						lable(for="profileImageURL") Picture Link:
```

Comment:
typo, should be label, also your for="" attribute should match the input id, which you dont have in your inputs yet. You have typos in other form elements here too